apps/condo/domains/acquiring/schema/AcquiringIntegration.test.js

```diff
@@ -2,71 +2,60 @@
  * Generated by `createschema acquiring.AcquiringIntegration 'name:Text;'`
  */
 
-import { expectToThrowGQLError } from '@open-condo/keystone/test.utils'
+import {
+    expectToThrowGQLError,
+    expectToThrowValidationFailureError,
+} from '@open-condo/keystone/test.utils'
+
+import {
+    SUPPORTED_BILLING_INTEGRATION_GROUP_DOESNT_EXIST_ERROR,
+} from '../constants/errors'
 
 const { makeLoggedInAdminClient, makeClient } = require('@open-condo/keystone/test.utils')
 const {
     expectToThrowAccessDeniedErrorToObj,
     expectToThrowAuthenticationErrorToObjects,
     expectToThrowAuthenticationErrorToObj,
-    expectToThrowValidationFailureError,
 } = require('@open-condo/keystone/test.utils')
 
-const { INTEGRATION_NO_BILLINGS_ERROR } = require('@condo/domains/acquiring/constants/errors')
 const { AcquiringIntegration, createTestAcquiringIntegration, updateTestAcquiringIntegration } = require('@condo/domains/acquiring/utils/testSchema')
-const { createTestBillingIntegration } = require('@condo/domains/billing/utils/testSchema')
 const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
 
 describe('AcquiringIntegration', () => {
     describe('CRUD tests', () => {
         describe('create',  () => {
             test('user can\'t', async () => {
-                const admin = await makeLoggedInAdminClient()
-                const [billingIntegration] = await createTestBillingIntegration(admin)
                 const client = await makeClientWithNewRegisteredAndLoggedInUser()
                 await expectToThrowAccessDeniedErrorToObj(async () => {
-                    await createTestAcquiringIntegration(client, [billingIntegration])
+                    await createTestAcquiringIntegration(client)
                 })
             })
             test('anonymous can\'t', async () => {
-                const admin = await makeLoggedInAdminClient()
-                const [billingIntegration] = await createTestBillingIntegration(admin)
                 const anonymousClient = await makeClient()
                 await expectToThrowAuthenticationErrorToObj(async () => {
-                    await createTestAcquiringIntegration(anonymousClient, [billingIntegration])
+                    await createTestAcquiringIntegration(anonymousClient)
                 })
             })
             test('support can', async () => {
                 const support = await makeClientWithSupportUser()
-                const [billingIntegration] = await createTestBillingIntegration(support)
-                const [integration, attrs] = await createTestAcquiringIntegration(support, [billingIntegration])
+                const [integration, attrs] = await createTestAcquiringIntegration(support)
                 expect(integration).toEqual(expect.objectContaining({
                     name: attrs.name,
-                    supportedBillingIntegrations: expect.arrayContaining([
```

Comment:
Add a single / multiple groups here, since it will be real-life scenario

Reply:
No multiple groups for now ...